lib/figo.rb

```diff
@@ -65,7 +64,7 @@ def query_api(path, data = nil)
       request.basic_auth(@client_id, @client_secret)
       request["Accept"] = "application/json"
       request["Content-Type"] = "application/x-www-form-urlencoded"
-      request["User-Agent"] =  "figo-ruby/1.3.1"
+      request["User-Agent"] =  "figo-ruby/1.4.0"
```

Comment:
Hmm… apparently this version has not been properly maintained throughout the project. The current version [seems to be 1.4.1](https://github.com/figo-connect/ruby-figo/blob/master/figo.gemspec#L3). See also [here](https://rubygems.org/gems/figo)